workflow/labels.md

```diff
@@ -0,0 +1,55 @@
+# Labels descriptions
+
+In this part you will find a description of all the labels we use on issues.
+
+## Labels
+
+| Label       | Description                                                        |
+| -----       | -----------                                                        |
+| Don’t merge | While this label is present this pull-request should not be merged |
+| Duplicate   | This issue already exists                                          |
+| Epic        |                                                                    |
+| Invalid     | The maintainers don’t agree with the issue                         |
+| Won’t fix   |                                                                    |
+
+## Priority
+
+| Label    | Description                                                        |
+| ----     | ---                                                                |
+| Critical | Has to be fixed ASAP                                               |
+| High     | This issue is causing a major regression, restricting functionally |
+| Medium   | This is not a damaging issue but requires attention in due course  |
+| Low      | Has been confirmed as a nice to have                               |
+
+## Review
+
+Each review label has 3 steps:
+* **-1:** Not approved
+* **Needed:** needs to get reviewed
+* **+1:** Approved
+
+| Label  | Description                     |
+| ----   | ---                             |
+| Code   | Validation steps of your code   |
+| Design | Validation steps of your design |
+| QA     | Validation steps of QA          |
+| UX     | Validation steps of UX          |
+
+## Status
+
+| Label   | Description                                                     |
+| ----    | ---                                                             |
+| Blocked | The issue is block by another issue (put it in the description) |
+| Doing   | Actually working on the issue                                   |
+| Review  | The issue is in review                                          |
+| Triaged |                                                                 |
+
+## Type
+
+| Label       | Description                                            |
+| ----        | ---                                                    |
+| Bug         | This issue is a bug                                    |
+| Enhancement | This issue improves the behavior, or adds new features |
+| Maintenance | Update to documentation or dependencies                |
+| Question    | To ask a question about something on the project       |
+| Takeover    |                                                        |
```

Comment:
"Takeover" seems overly specific - a takeover would just be an enhancement, right?

Reply:
Please remove Takeover